src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,29 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
```

Comment:
This is an interesting approach, I like the way youve split this into 2 functions to separate out the species. In this case it works fine, but its worth thinking about what this would look like should the functionality need to change, for example bringing in another species. You couldnt pass in a boolean to determine the species then, something like a switch statement would be needed to build that in there.

This isnt a criticism btw, just an observation of something you may have to consider in the future when writing your functions